src/components/Board/Navbar/Navbar.css

```
@@ -60,3 +60,19 @@
 .Navbar .LoginSignUpButton:focus {
   color: #fff;
 }
+
+@media (max-width: 768px) {
+  .Navbar__group--end .MuiIconButton-root,
```

I think you can remove all the css lines under this line if you you use 'padding-inline' instead of 'padding'
see:
```suggestion
  .Navbar__group .MuiIconButton-root,
  .Navbar__group .MuiIconButton-root {
    padding-inline: 6px;
  }
```
Let me know if i am wrong please :)